src/test/java/plugins/SubversionPluginTest.java

```diff
@@ -49,7 +54,8 @@ public void run_basic_subversion_build() throws SubversionPluginTestException {
         f.addShellStep("test -d .svn");
         f.save();
 
-        f.startBuild().shouldSucceed().shouldContainsConsoleOutput("test -d .svn");
+        Build b = f.startBuild().shouldSucceed();
+        assertThat(b.getConsole(), Matchers.containsRegexp("test -d .svn", Pattern.MULTILINE));
```

Comment:
:ant: This will match, for example, `test -d xsvn`. I think you wanted to use `Matchers.containsString`.